browser/chromium/chromium_darwin.go

```diff
@@ -24,6 +25,24 @@ var (
 func (c *Chromium) GetMasterKey() ([]byte, error) {
 	// don't need chromium key file for macOS
 	defer os.Remove(types.ChromiumKey.TempFilename())
+
+	// Try get the master key via CVE-2025-24204
+	if cve2025_24204.GetMacOSVersion() == cve2025_24204.ExploitOSVersion {
```

Comment:
The impact scope of the vulnerability is `15.x`; can a prefix starting with `15` be used to determine it?